client/my-sites/marketplace/test/test.constants.ts

```diff
@@ -0,0 +1,64 @@
+/**
+ * External dependencies
+ */
+import { YOAST_PREMIUM, YOAST_FREE } from '@automattic/calypso-products';
+/**
+ * Internal dependencies
+ */
+import {
+	getPluginsToInstall,
+	getDefaultPluginInProduct,
+	getDefaultProductInProductGroup,
+	getProductDefinition,
+	productGroups,
+} from '../constants';
+import { YOAST } from '../types';
+
+describe( 'Constants and product related utilities test', () => {
```

Comment:
See, it feels a bit odd to be testing constants in the first place. So we're either not testing properly, or these aren't constants and should live in another file. 😉